### ISSUE TITLE
Moving-boundary problems

### ISSUE THREAD

**Issue by cubayang** [lululxvi/deepxde#452]
Dear Lulu,

Thank you for your amazing work.

I am new to deepxde and I have two questions after trying a few examples:

1. Is it possible to use deepxde to solve moving-boundary problems?
2. Since the PDE is incorporated into the loss function, does that mean deepxde requires fewer training data than the vanilla FNN to achieve the same accuracy?

Thanks in advance.

**Comment by lululxvi** [comment]
Yes to both.
